sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewTestSuite.scala

```diff
@@ -200,6 +200,20 @@ abstract class SQLViewTestSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("creating local temp view should not affect existing table reference") {
```

Comment:
this doesn't test different view type, seems we will just it repeatedly for no good.

Reply:
sorry, I messed the code up after rebasing. fixed

Reply:
hmm, the test is still here...